src/Icons/phpunit.xml.dist

```diff
@@ -1,31 +1,31 @@
 <?xml version="1.0" encoding="UTF-8"?>
-
 <!-- https://phpunit.de/manual/current/en/appendixes.configuration.html -->
-<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:noNamespaceSchemaLocation="vendor/bin/.phpunit/phpunit.xsd"
-         colors="true"
-         bootstrap="tests/bootstrap.php"
-         failOnRisky="true"
-         failOnWarning="true"
+<phpunit
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="vendor/bin/.phpunit/phpunit.xsd"
+        colors="true"
+        bootstrap="vendor/autoload.php"
```

Comment:
not the same file ?

Reply:
Nice catch, I will double-check later today